orchestrator/orchestrator.go

```diff
@@ -215,7 +215,7 @@ func (s *Orchestrator) processTask(task *process.Process_Node_Task, wf *process.
 		ProcessHash:  wf.Hash,
 		EventHash:    eventHash,
 		ParentHash:   execHash,
-		StepID:       task.Key,
+		StepID:       nodeKey,
```

Comment:
we should rename execution's StepID to stepKey.
@antho1404 what do you think?

Reply:
I'm ok to change it, but why not `nodeKey` like that we always talk about the same thing.

Reply:
`nodeKey` 👍